### ISSUE TITLE
onDateRender() info incorrect

### ISSUE THREAD

**Issue by MarekTVGla** [GramThanos/jsCalendar#45]
your calculations for isCurrentMonth, isPreviousMonth, isNextMonth are off by 1, so the first day of the month's info shows it as part of the previous month.

You just need to change your evaluations for i to (i+1) on lines 795-797.

**Comment by GramThanos** [comment]
Thanks @MarekTVGla 😄

**Comment by fabiangigler** [comment]
@GramThanos I Just noticed a possibly related bug on the onMonthRender info.
The info.start date seems to be always off by one as well.

Code:
```
calendar.onMonthRender(function(date, element, info) {
    console.log( info );
});
```

Console:
![grafik](https://user-images.githubusercontent.com/3050232/68675358-fe011980-0557-11ea-88d0-cba4220e4b0b.png)

**Comment by GramThanos** [comment]
Yeah, that bug exists on the latest release, but it was patched on the development version #43
